app/models/event_registration.rb

```diff
@@ -1,7 +1,6 @@
 class EventRegistration < ApplicationRecord
+  belongs_to :user
```

Comment:
similar here, and maybe make this something like: `belongs_to :registrant, class_name: "User", foreign_key: :registrant_id`